android/demo/build.gradle

```diff
@@ -1,37 +1,27 @@
 buildscript {
   dependencies {
-    classpath 'com.android.tools.build:gradle:2.1.3'
+    classpath 'com.android.tools.build:gradle:2.2.3'
   }
 }
 
 apply plugin: 'com.android.application'
 
 android {
-  compileSdkVersion 24
-  buildToolsVersion '24.0.2'
+  compileSdkVersion 25
+  buildToolsVersion '25.0.0'
 
   defaultConfig {
     minSdkVersion 15
-    targetSdkVersion 24
+    targetSdkVersion 25
   }
 
   sourceSets.main {
-    manifest.srcFile 'AndroidManifest.xml'
-    java.srcDirs = ['src']
-    resources.srcDirs = ['src']
-    res.srcDirs = ['res']
     assets.srcDirs = ['../../scenes']
   }
-  buildTypes {
-    debug {
-      // applicationIdSuffix ".debug"
-      jniDebuggable true
```

Comment:
This is a good point about the "debuggable" flags, I'm not sure of the impact of these flags on either the library or demo module under the new build system, but I can try both and see.